app/src/main/java/pm/gnosis/heimdall/ui/settings/SettingsActivity.kt

```diff
@@ -39,6 +39,7 @@ class SettingsActivity : BaseActivity() {
             startActivity(TokenManagementActivity.createIntent(this))
         }
 
+        // TODO: this check should be done in the SecuritySettingsActivity
```

Comment:
Actually this is not correct in the current state 😬 

We added this when we only had the fingerprint setting. But we added the "Reveal Security Phrase" and "Change Password" later on... So these options are only accessible on devices with fingerprint.

Reply:
I opened a ticket https://gnosisltd.atlassian.net/browse/WA-111